docs/annotation.md

```diff
@@ -36,7 +54,7 @@ for crop in output_crops:
     assert os.path.exists(crop)
```

Comment:
I suggest dropping the assert statements here. Typically the user just wants to know the commands, not have proof that they work and a lot of beginners won't really understand what's going on with assert. If we want something that shows the files were created then I think this would be clearer for most novices:

```python
import os

os.listdir()
```